test/unit/sso/openid_connect_test.rb

```diff
@@ -77,6 +77,20 @@ class OpenidConnectTest < ActiveSupport::TestCase
         assert_equal users(:one), subject.authenticated?
       end
     end
+
+    test "it accepts group paratameter in payload and authenticates the user" do
+      oidc_source = AuthSourceExternal.where(:name => 'External').first_or_create
+      external = FactoryBot.create(:external_usergroup, :auth_source => oidc_source)
+      usergroup = FactoryBot.create(:usergroup, :admin => true)
+      external.update(:usergroup => usergroup, :name => usergroup.name)
+
+      User.current = nil
+      payload['groups'] = usergroup.name
```

Comment:
i think this should be:
```suggestion
      payload['groups'] = [usergroup.name]
```
if my understanding of the docs is correct